src/RecoveryServices/RecoveryServices/Common/PSRecoveryServicesVaultClient.cs

```diff
@@ -99,9 +100,9 @@ public List<Microsoft.Azure.Management.Internal.Resources.Models.ResourceGroup>
         /// <param name="vaultStorageUpdateRequest">Backup Properties Update</param>  
         /// <returns>Azure Operation response object.</returns>  
         public void UpdateVaultStorageType(string resouceGroupName, string vaultName,
-            BackupStorageConfig backupStorageConfig)
+            BackupResourceConfigResource backupStorageConfig)
```

Comment:
Please try to use BackupResourceConfig as the replacement for BackupStorageConfig being used earlier

Reply:
> Please try to use BackupResourceConfig as the replacement for BackupStorageConfig being used earlier

BackupResourceConfigResource contains  BackupResourceConfig as properties field and using it we are referring BackupResourceConfig.

Reply:
Okay, your changes look fine